SAGE Journals.js

```diff
@@ -131,10 +131,16 @@ function scrape(doc, url) {
 				});
 			}
 
-			var tagentry = ZU.xpathText(doc, '//kwd-group[1] | //*[contains(concat( " ", @class, " " ), concat( " ", "hlFld-KeywordText", " " ))]');
+            // Only get attributes and skip introductory term Keywords
+			var tagentry = ZU.xpathText(doc, '//kwd-group[1] | //*[contains(concat( " ", @class, " " ), concat( " ", "hlFld-KeywordText", " " ))] \
```

Comment:
I guess this could be declared with `let` because it's not used in a lambda function (same for `rows` variable)